.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform.lock.hcl
+.terraform/
+terraform.tfstate
+terraform.tfstate.backup
```

Comment:
could you please, add a newline at the end of file.

Reply:
I have added